airline-data/src/main/scala/com/patson/model/airplane/Model.scala

```diff
@@ -81,15 +81,21 @@ object Model {
                       Model("Bombardier Q400", capacity = 90, fuelBurn = (90 * 3.5).toInt, speed = 556, range = 2040, price = 35000000, lifespan = 30 * 52, constructionTime = 8, countryCode = "CA", imageUrl = "https://www.norebbo.com/2015/08/bombardier-dhc-8-402-q400-blank-illustration-templates/"),
+                      Model("Embraer EMB190", capacity = 100, fuelBurn = (100 * 3.3).toInt, speed = 823, range = 4537, price = 60000000, lifespan = 30 * 52, constructionTime = 8, countryCode = "BR", imageUrl = "https://www.norebbo.com/2015/06/embraer-190-blank-illustration-templates/"),
                       Model("Sukhoi Superjet 100", capacity = 108, fuelBurn = (108 * 4.1).toInt, speed = 828, range = 4578, price = 40000000, lifespan = 30 * 52, constructionTime = 8, countryCode = "RU", imageUrl = "https://www.norebbo.com/2016/02/sukhoi-ssj-100-blank-illustration-templates/"),
+                      Model("Fokker 100", capacity = 109, fuelBurn = (109 * 3.6).toInt, speed = 845, range = 3170, price = 55000000, lifespan = 30 * 52, constructionTime = 8, countryCode = "NL", imageUrl = "https://www.norebbo.com/2018/07/fokker-100-f-28-0100-blank-illustration-templates/"),
                       Model("Airbus A318", capacity = 132, fuelBurn = (132 * 3).toInt, speed = 829, range = 7800, price = 90000000, lifespan = 35 * 52, constructionTime = 8, countryCode = "NL", imageUrl = "https://www.norebbo.com/2018/01/airbus-a318-blank-illustration-templates-with-pratt-whitney-and-cfm56-engines/"),
                       Model("Bombardier CS100", capacity = 133, fuelBurn = (133 * 3.3).toInt, speed = 828, range = 5741, price = 80000000, lifespan = 35 * 52, constructionTime = 8, countryCode = "CA", imageUrl = "https://www.norebbo.com/2016/02/bombardier-cs100-blank-illustration-templates/"),
                       Model("Boeing 737-700C", capacity = 140, fuelBurn = (140 * 3.3).toInt, speed = 825, range = 6083, price = 85000000, lifespan = 35 * 52, constructionTime = 12, countryCode = "US", imageUrl = "https://www.norebbo.com/2014/04/boeing-737-700-blank-illustration-templates/"),
+                      Model("McDonnel Douglas MD-90", capacity = 160, fuelBurn = (160 * 3.55).toInt, speed = 811, range = 3787, price = 80000000, lifespan = 30 * 52, constructionTime = 8, countryCode = "US", imageUrl = "https://www.norebbo.com/2018/02/mcdonnell-douglas-md-90-blank-illustration-templates/"),
                       Model("Boeing 737-800", capacity = 184, fuelBurn = (184 * 3.8).toInt, speed = 825, range = 5436, price = 100000000, lifespan = 35 * 52, constructionTime = 24, countryCode = "US", imageUrl = "https://www.norebbo.com/2012/11/boeing-737-800-blank-illustration-templates/"),
                       Model("Airbus A320neo", capacity = 195, fuelBurn = (195 * 4).toInt, speed = 833, range = 6500, price = 110000000, lifespan = 35 * 52, constructionTime = 8, countryCode = "NL", imageUrl = "https://www.norebbo.com/2017/08/airbus-a320-neo-blank-illustration-templates/"),
```

Comment:
This line is my bad...construction time should be 24 more aligned with other planes of the same size - im going to change this on my side

Reply:
That's for you to decide, I didn't want to mess with the system you had established. 
But yeah, it was supposed to be 24, I think i forgot to change, sorry.